Sources/StreamChatUI/CommonViews/InputTextView/InputTextView.swift

```diff
@@ -5,9 +5,24 @@
 import StreamChat
 import UIKit
 
+/// The delegate of the `InputTextView` that notifies when an attachment is pasted in the text view.
+public protocol InputTextViewClipboardAttachmentDelegate: AnyObject {
+    /// Notifies that an `UIImage` has been pasted into the text view
+    /// - Parameters:
+    ///   - inputTextView: The `InputTextView` in which the image was pasted
+    ///   - image: The `UIImage`
+    func inputTextView(_ inputTextView: InputTextView, didPasteImage image: UIImage)
+}
+
 /// A view for inputting text with placeholder support. Since it is a subclass
 /// of `UITextView`, the `UITextViewDelegate` can be used to observe text changes.
 open class InputTextView: UITextView, AppearanceProvider {
+    /// The delegate which gets notified when an attachment is pasted into the text view
+    open var clipboardAttachmentDelegate: InputTextViewClipboardAttachmentDelegate?
+    
+    /// Whether this text view should allow images to be pasted
+    open var allowsPastingImages: Bool = true
```

Comment:
I think we should try to make our boolean properties read as assertions like the Swift Style Guide, we already have a pattern for enabling/disabled things:
```suggestion
    /// Whether this text view should allow images to be pasted
    open var isPastingImagesEnabled: Bool = true
```

Reply:
Changed this according to the Swift Style guide